PendingReleaseNotes

```diff
@@ -50,13 +50,21 @@
   recommend that users with versioned buckets, especially those that existed
   on prior releases, use these new tools to check whether their buckets are
   affected and to clean them up accordingly.
+* CephFS: Running the command "ceph fs authorize" for an existing entity now
+  upgrades the entity's capabilities instead of printing an error. It can now
+  also change read/write permissions in a capability that the entity already
+  holds. If the capability passed by user is same as one of the capabilities
+  that the entity already holds, idempotency is maintained.
 * CephFS: Two FS names can now be swapped, optionally along with their IDs,
   using "ceph fs swap" command. The function of this API is to facilitate
   file system swaps for disaster recovery. In particular, it avoids situations
   where a named file system is temporarily missing which would prompt a higher
   level storage operator (like Rook) to recreate the missing file system.
   See https://docs.ceph.com/en/latest/cephfs/administration/#file-systems
   docs for more information.
+* CephFS: Before running the command "ceph fs rename", the CephFS to be
```

Comment:
Before running the command "ceph fs rename", the filesystem to be renamed must be offline and the config "refuse_client_session" must be set.

(also, note when the config should be unset).